packages/common/src/dto/placement-preferences.dto.ts

```diff
@@ -1,6 +1,13 @@
 import { IsBoolean, IsOptional } from 'class-validator';
 
 export class PlacementPreferencesDTO {
```

Comment:
Should sub-objects be DTOs as well? I like it, we might need to update the specialty objects in 3rd step to be more like this